integration-tests/src/test/scala/eu/neverblink/jelly/integration_tests/rdf/ProtocolSpec.scala

```diff
@@ -0,0 +1,223 @@
+package eu.neverblink.jelly.integration_tests.rdf
+
+import eu.neverblink.jelly.convert.jena.traits.JenaTest
+import eu.neverblink.jelly.core.proto.v1.{PhysicalStreamType, RdfStreamFrame, RdfStreamOptions}
+import eu.neverblink.jelly.integration_tests.rdf.io.*
+import eu.neverblink.jelly.integration_tests.util.JellyCli
+import eu.neverblink.jelly.integration_tests.util.OrderedRdfCompare
+import eu.neverblink.jelly.integration_tests.util.ProtocolTestVocabulary.*
+import org.apache.jena.rdf.model.{ModelFactory, Resource}
+import org.apache.pekko.actor.ActorSystem
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.io.{File, FileInputStream}
+import java.util.UUID.randomUUID
+
+class ProtocolSpec extends AnyWordSpec, Matchers, ScalaFutures, JenaTest:
+  given ActorSystem = ActorSystem("test")
+
+  val frameSize = 10
+
+  runSerializationTest(JenaStreamSerDes)
+  runSerializationTest(Rdf4jSerDes)
+  runSerializationTest(Rdf4jReactiveSerDes())
+  runSerializationTest(JenaReactiveSerDes())
+  // TODO: Reenable Titanium
+  // runSerializationTest(TitaniumSerDes)
+
+  runDeserializationTest(JenaStreamSerDes)
+  runDeserializationTest(Rdf4jSerDes)
+  runDeserializationTest(Rdf4jReactiveSerDes())
+  runDeserializationTest(JenaReactiveSerDes())
+  // TODO: Reenable Titanium
+  // runDeserializationTest(TitaniumSerDes)
+
+  private def runSerializationTest[TNSer, TTSer, TQSer](ser: ProtocolSerDes[TNSer, TTSer, TQSer]): Unit =
```

Comment:
suggest renaming to `runRdfToJellyTest` to be consistent with the names of the test cases